`flake.nix` missing `vusted` dependency

### Description

The nix flake doesn't contain all dependencies required to run Makefile.

```
$ nix develop
$ make
vusted ./test
make: vusted: No such file or directory
make: *** [Makefile:2: test] Error 127
```

### Neovim version

NVIM v0.9.2
Build type: Release
LuaJIT 2.1.1693350652

### Nvim-lspconfig version

_No response_

### Operating system and version

macOS 13.4.1

### Affected language servers

all

### Steps to reproduce

See description

### Actual behavior

See description

### Expected behavior

_No response_

### Minimal config

```Lua
n/a
```


### LSP log

n/a

--- a/flake.nix
+++ b/flake.nix
@@ -1,6 +1,7 @@
 {
   description = "Quickstart configurations for the Nvim LSP client";
 
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs";
   inputs.flake-utils.url = "github:numtide/flake-utils";
 
   outputs = { self, nixpkgs, flake-utils }:
@@ -11,6 +12,7 @@
           buildInputs = [
             pkgs.stylua
             pkgs.luaPackages.luacheck
+            pkgs.luajitPackages.vusted
             pkgs.selene
           ];
         };

